admin/repository/prebuild-musicbrainz

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env bash
+
+set -e -u
+
+# shellcheck source=admin/lib/common.inc.bash
+source "$(dirname "${BASH_SOURCE[0]}")/../lib/common.inc.bash"
+
+HELP=$(cat <<EOH
+Usage: $SCRIPT_NAME
+   or: $SCRIPT_NAME --help
+
+Build a new image for the musicbrainz service
+from the MusicBrainz Server source code repository
+and push it as metabrainz/musicbrainz-docker-musicbrainz:<tag>
+where tag is based on MusicBrainz Server version a build sequence number.
```

Comment:
Should that be "version and a build sequence number"?

Reply:
Good catch, fixed! See https://github.com/metabrainz/musicbrainz-docker/compare/255a6c15ea3fd4e946544f8d8436c2164a65c48f..c8cf07773d6d69beb4d456db6bd0dbf4c203266e